install.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+echo 'vm.max_map_count=262144' | sudo tee --append /etc/sysctl.d/99-elasticsearch.conf
```

Comment:
This only works if you are doing a local install - see REMOTE_CLUSTER - also, you have to apply it to every elasticsearch node in the cluster